peachjam/settings.py

```diff
@@ -151,16 +153,20 @@
 ]
 # admins must create accounts
 ACCOUNT_SIGNUP_ENABLED = False
-# sign in with email addresses
-ACCOUNT_AUTHENTICATION_METHOD = "email"
 # email addresses are required for new accounts
 ACCOUNT_EMAIL_REQUIRED = True
 ACCOUNT_PRESERVE_USERNAME_CASING = False
 ACCOUNT_SESSION_REMEMBER = True
 ACCOUNT_EMAIL_SUBJECT_PREFIX = EMAIL_SUBJECT_PREFIX
-ACCOUNT_DEFAULT_HTTP_PROTOCOL = "https"
+ACCOUNT_DEFAULT_HTTP_PROTOCOL = "http" if DEBUG else "https"
 LOGIN_URL = "account_login"
 LOGIN_REDIRECT_URL = "home_page"
+ACCOUNT_EMAIL_VERIFICATION = "mandatory"
```

Comment:
This may be a problem, because it forces all accounts to be verified before a user can log in, including admins.

Reply:
should we disable this then?

Reply:
or force admins to verify themselves

Reply:
so admins will still be able to login to the admin site, but can't login via the all auth login pages

Reply:
let's keep it like this and change if necessary